examples/api/lib/widgets/scrollbar/raw_scrollbar.shape.0.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
is adding this intentional?

Reply:
Yes, these are examples that snuck into the tree as embedded examples instead of linked examples, I'm just moving them out. I'm planning on adding a lint that will stop people from checking in `dartpad` and `sample` embedded samples once I've extracted them from dart:ui (the only remaining embedded non-"snippet" samples).